### PR TITLE
Feature/66550 restyle datepicker

### PR DESCRIPTION
**Pull request by trishasalas**
https://central.tri.be/issues/66550

_related:_ https://github.com/moderntribe/the-events-calendar/pull/1117

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,10 @@ vendor/*
 !vendor/lucatume/di52
 !vendor/xrstf/composer-php52
 tests/_output/*
+/.DS_Store
+/src/.DS_Store
+/src/resources/.DS_Store
+/src/resources/postcss/.DS_Store
+/src/resources/images/.DS_Store
+/src/resources/postcss/tec.sublime-project
+/src/resources/postcss/tec.sublime-workspace
```
Comment (borkweb) [inline .gitignore]: These additions should be added to your global .gitignore rather than in the repo-specific one

File: src/Tribe/Main.php [hunk 2]
```diff
@@ -172,8 +172,7 @@ public function load_assets() {
 				array( 'tribe-dependency', 'dependency.js', array( 'jquery', 'underscore' ) ),
 				array( 'tribe-dependency-style', 'dependency.css' ),
 				array( 'tribe-pue-notices', 'pue-notices.js', array( 'jquery' ) ),
-				array( 'tribe-jquery-ui-theme', 'vendor/jquery/ui.theme.css' ),
-				array( 'tribe-jquery-ui-datepicker', 'vendor/jquery/ui.datepicker.css' ),
+				array( 'tribe-datepicker', 'datepicker.css' ),
```
Comment (barryhughes) [inline src/Tribe/Main.php]: Are those vendor libs required any place else? If they were only being used in relation to the previous datepicker, should we now purge them?

Comment (barryhughes) [inline src/Tribe/Main.php]: Sounds good

**Comment by barryhughes** [comment]
:taco:

**Pull request merged**